src/_includes/partials/anonymous-contact-form.njk

```diff
@@ -0,0 +1,22 @@
+<header class="content__header">
+  <h1 class="title page__title">
+    {{ title }}
+  </h1>
+</header>
+
+<article class="bg copy page__content">
+  {{ content | markdown | safe }}
+
+  <br>
+
+  <form name="coc-report" method="POST" data-netlify="true" action="/success-message">
+    <p>
```

Comment:
Probably worth adding the bot-trapping [honeypot input](https://www.netlify.com/docs/form-handling/#spam-filtering) too